index.js

```diff
@@ -91,7 +149,14 @@ const promo20 = users => {
 // ***************************************************************************
 
 const addActive = users => {
-  // Your code goes here ...
+  const activeUsers = [];
+
+  for (let user of users) {
+    user.isActive = true
+    activeUsers.push(user);
+  }
+  
```

Comment:
This function immediately is changing the values you are passing onto it.

In functions, unless said otherwise, try to create a habit that you return a totally new value.
For example
```js
const addActive = users => {
return users.map(user => {
return {
...user, //here you are getting the whole user object copied
isActive: true
}
})
}
```
After this function you get an array of every user active but you get a totally new value.